src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -41,7 +41,7 @@
     }
   },
   "packs": {
-    "Microsoft.Android.Sdk": {
+    "Microsoft.Android.Sdk.net6": {
```

Comment:
I guess renaming this breaks Xamarin.Legacy.Sdk because of this line:

https://github.com/xamarin/Xamarin.Legacy.Sdk/blob/ce48ad6d67a77c7a05bcf31155a033a3539a0ff4/src/Xamarin.Legacy.Sdk/Sdk/Xamarin.Legacy.Android.targets#L28

Is there a way to also keep the old name and alias to `Microsoft.Android.Sdk.net6`?

Reply:
Ok, I pushed: https://www.nuget.org/packages/Xamarin.Legacy.Sdk/0.2.0-alpha2

I don't know how long it takes to show up on `dotnet-public`:

https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-public/NuGet/Xamarin.Legacy.Sdk

When it does, we can  update this line:

https://github.com/xamarin/xamarin-android/blob/a689fb9b0e1459c95115b4c9949cb7ec4068f2be/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/XASdkTests.cs#L931

Reply:
Updated the tests to use the new version, that PR should fix those failures.